packages/babel-plugin-transform-parameters/test/fixtures/parameters/default-arguments/exec.js

```diff
@@ -0,0 +1,4 @@
+function func(...arguments) {
```

Comment:
Can you also add a test like `functino func(a, b, ...arguments)`?

Reply:
Yeah sure, I will add a test a file with these arguments.

Reply:
Both in strict and non-strict mode please, since `arguments` behaves differently!

Reply:
Is this new test looks good?

Reply:
Could you return `{ a, b, arguments }` instead in the exec test?

What I was trying to test is that if we accidentally stop emitting `arguments = new Array` (because `arguments` is already implicitly defined), `a` and `b` would be accidentally reassigned. It's not a bug in this PR, but while reviewing it I though "oh, we must be careful about it!" so it's worth adding a test :slightly_smiling_face: 

Btw, you should run `OVERWRITE=true yarn jest` to fix the test failures reported by CI.

Reply:
Okay, I will do the same for this.

Reply:
Does this test change looks good?

Reply:
I wrote `OVERWRITE=true yarn jest` the test were passing locally but in CI they just fail, I don't know what the problem is?

Reply:
Uhm, maybe try to debase this branch on `main` and re-run `make bootstrap`.

Reply:
I got the problem, It's working now, Thank you!